### PR TITLE
refactor: changed to DB Screen Flex (variable font)

### PR DESCRIPTION
**Pull request by mfranzke**
## Proposed changes

Resolves https://github.com/db-ui/mono/issues/2087

- [ ] add it to the new files out of https://github.com/db-ui/mono/pull/2028 as well
- [x] ~using [`tech(variations)`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/src#font_technologies) to differentiate the variable font by further provided regular font files for downwards compatibility~ (not necessary, compare to https://caniuse.com/variable-fonts)

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: .github/workflows/02-e2e-showcases.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
       fail-fast: false
       matrix:
         framework: [angular, react, vue]
-        shard: [1/2, 2/2]
+        shard: [1of2, 2of2]
```
Comment (nmerget) [inline .github/workflows/02-e2e-showcases.yml:20]: These changes should be moved to another branch I guess

Comment (mfranzke) [reply .github/workflows/02-e2e-showcases.yml:20]: yolo

Comment (mfranzke) [reply .github/workflows/02-e2e-showcases.yml:20]: seriously: I thought that I've already done that. For whatever reason I didn't.

Comment (mfranzke) [reply .github/workflows/02-e2e-showcases.yml:20]: https://github.com/db-ui/mono/pull/2150

**Pull request closed without merging**